scripts/DB/sample_data.sql

```diff
@@ -52,3 +52,18 @@ INSERT INTO "GroupMemberships" ("UserId", "GroupId") VALUES
 ( 7,  14),
 ( 7,   1),
 ( 5,  14);
+
+INSERT INTO "Projects" ("Name", "TypeId", "Description", "OwnerId", "GroupId", "OrganizationId", "Language","Private", "DateCreated", "DateUpdated", "DateArchived", "AllowDownloads", "AutomaticBuilds") VALUES
+( 'Developer Town Project 1', 1, 'The Sogdian Bible was translated by Mike Davis and his team that has been working in Central Asia since 1986. The full New Testament translation was completed in 2018 and has been made open source for all to use and enjoy.', 5, 14, 2, 'English', false, now(), now(),null, true, true),
```

Comment:
This is probably fine for test data.  I can modify them later to get more realistic samples.